persistence/indexer/indexer.go

```diff
@@ -148,15 +148,15 @@ func (indexer *txIndexer) indexByHeightAndIndex(height int64, index int32, bz []
 	return indexer.db.Set(indexer.heightAndIndexKey(height, index), bz)
 }
 
-func (indexer *txIndexer) indexBySender(sender string, bz []byte) error {
-	return indexer.db.Set(indexer.senderKey(sender), bz)
+func (indexer *txIndexer) indexBySenderHeightAndIndex(sender string, height int64, index int32, bz []byte) error {
```

Comment:
```suggestion
func (indexer *txIndexer) indexBySenderHeightAndBlockIndex(sender string, height int64, blockIndex int32, bz []byte) error {
```

Reply:
Ditto everywhere else. Too many `index` in one line and I want to make sure its clear